docs/api/1.0.0-SNAPSHOT.md

```diff
@@ -0,0 +1,122 @@
+# API Docs - v1.0.0-SNAPSHOT
```

Comment:
Remove this file, this is auto-generated at release time

Reply:
Fixed with commit: `Fix some suggested issues in PR#1.` 8a89f22